.goreleaser.yml

```diff
@@ -240,8 +240,9 @@ dockers:
       - "--label=version={{.Version}}"
       - "--platform=linux/amd64"
     extra_files:
-      - plugins
-      - config/example.yaml
+      - release_deps/VERSION.txt
+      - release_deps/plugins
+      - release_deps/config.yaml
       - config/logging.stdout.yaml
```

Comment:
Do you think it's worthwhile to copy over `logging.stdout.yaml` into the release_deps folder to follow the same principle or leave it as is? I don't know that I have a preference, just not sure what makes sense here

Reply:
I am indifferent. It is not being modified or generated. It is only used by the container build process.